### ISSUE TITLE
Add Audio Feature Extraction Task

### ISSUE THREAD

**Issue by ecyht2** [huggingface/huggingface.js#921]
Currently, the `Feature Extraction` task includes both models for audio and text feature extraction (it is officially placed under the NLP modality). I think it would be nice to have a new task for `Audio Feature Extraction` task just like `Image Feature Extraction` to better label models. 

Some `Audio Feature Extraction` models in `Feature Extractions`:

- [facebook/encodec](https://huggingface.co/facebook/encodec_24khz)
- [kyutai/mimi](https://huggingface.co/kyutai/mimi)

**Comment by osanseviero** [comment]
Yes, having an `audio-feature-extraction` has been discussed few times in the past and there's agreement having it could make sense. cc @Vaibhavs10 @pcuenca

**Comment by merveenoyan** [comment]
I think it makes sense to have given we have a separate one for image too @Vaibhavs10

**Comment by Vaibhavs10** [comment]
Hey @ecyht2 - sorry for the delay in responding to you - this is in the works, we're in the process of adding this as a recognised task in transformers and then follow it up with a task page. Cheers!
